media/css/firefox/welcome.scss

```diff
@@ -71,14 +71,15 @@ $image-path: '/media/protocol/img';
     }
 }
 
-.mzp-c-hero-title {
+.mzp-c-hero-title, .hero-subheading {
```

Comment:
I don't think the `.hero-subheading` class is used. Is this is a leftover?

Reply:
yep you are totally right, I was trying something and missed taking that bit out!